client/src/views/mobile/CellMobileView.tsx

```diff
@@ -38,97 +40,140 @@ const CellMobileView = ({
   confirmReport,
   confirmSacrifice,
   confirmReset,
+  bruteList
 }: CellMobileViewProps) => {
   const { t } = useTranslation();
   const { brute, owner } = useBrute();
+  const navigate = useNavigate();
   const theme = useTheme();
   const isXs = useMediaQuery(theme.breakpoints.only('xs'));
+  const pageRef = useRef<HTMLInputElement>(null);
+
+  useEffect(
+    () => {
+      let touchstartX = 0;
+      let touchendX = 0;
+      const checkSwipe = () => {
+        if (touchendX > touchstartX && touchendX - touchstartX > -120) {
+          if (bruteList && bruteList[bruteList.indexOf(brute!.name) - 1]) {
+            navigate(`/${bruteList[bruteList.indexOf(brute!.name) - 1]}/cell`);
+          }
+        }
+        if (touchendX < touchstartX && touchstartX - touchendX > -120) {
+          if (bruteList && bruteList[bruteList.indexOf(brute!.name) + 1]) {
+            navigate(`/${bruteList[bruteList.indexOf(brute!.name) + 1]}/cell`);
+          }
+        }
+      };
+
+      document.addEventListener('touchstart', (e) => {
+        touchstartX = e.changedTouches[0].screenX;
+      }, false);
+      document.addEventListener('touchmove', (e) => {
+        touchendX = e.changedTouches[0].screenX;
+        checkSwipe();
+      }, false);
+
+      return () => {
+        document.removeEventListener('touchstart', (e) => {
+          touchstartX = e.changedTouches[0].screenX;
+        });
+        document.removeEventListener('touchmove', (e) => {
+          touchendX = e.changedTouches[0].screenX;
+          checkSwipe();
+        });
```

Comment:
Use the handlers stored in my previous comment, instead of creating new ones, since this won't pass the reference equality check